CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master
 
+* Added `onStubMissing` to report missing stubs
```

Comment:
Hey @c1ira could you add a period + 2 spaces at the end of this line? It's a little trick in markdown so that once rendered, it inserts a new line inside the same paragraph (see other entries)

Also don't hesitate to add a link to that PR too (under the link to your name) while you're at it